lib/concurrent-ruby/concurrent/utility/processor_counter.rb

```diff
@@ -21,6 +22,23 @@ def physical_processor_count
         @physical_processor_count.value
       end
 
+      def available_processor_count
+        cpu_count = processor_count.to_f
+        quota = cpu_quota
```

Comment:
Shouldn't we multiply those? (like mentioned in https://github.com/ruby-concurrency/concurrent-ruby/pull/1035#discussion_r1469717571)

Reply:
Nah, that was a mistake on my part. The cgroup quota is expressed in how many microsecond of CPU time you can use over a given period (also in micro-second).

So if you use docker with something like `--cpus=2.5`, the `cpu.max` will be something like `250000000 10000000`, so when we divide them we get our `2.5` back.